### PR TITLE
Created Authenticated Callbacks Guide

### PR DESCRIPTION
**Pull request by ajrice6713**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-1345.s3-website-us-east-1.amazonaws.com/index.html

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-1345.s3-website-us-east-1.amazonaws.com/index.html

File: guides/callbacks/callbacks.md [hunk 1]
```diff
@@ -24,12 +24,26 @@ If you need static IPs for your callbacks, feel free to open up a ticket on our
...
 
 <br>
+
+## Authenticated Callbacks {#authenticatedCallbacks}
+
+Bandwidth allows you to enable basic authentication on the webhooks received from the different services. For messaging and voice, credentials would be set on the messaging or voice application created in the Bandwidth Dashboard. In a subscription, the basic authentication credentials would be set in the <code class="post">POST</code> request to create each specific subscription type. To authenticate, Bandwidth follows the basic HTTP authentication framework outlined in [RFC 7235](https://tools.ietf.org/html/rfc7235).
+
+Essentially, if you have basic authentication enabled for a webhook, Bandwidth will send a request with no authorization header attached and expect a 401 response that includes a `WWW-AUTHENTICATE` header containing a challenge. Once received, Bandwidth will send a second request that includes the authorization header and webhook body.
+
+Bandwidth uses basic authentication, so an example of that header could look like `WWW-AUTHENTICATE: Basic realm=""`.
```
Comment (jmulford-bw) [inline guides/callbacks/callbacks.md]: You could maybe clarify that this is what bandwidth sends. The paragraph break kinda loses the context IMO

**Review by jmulford-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-1345.s3-website-us-east-1.amazonaws.com/index.html

**Pull request merged**